index.html

```diff
@@ -980,7 +1003,7 @@ <h4>ethereumAddress</h4>
                 <a href="https://w3id.org/security#ethereumAddress">security-vocab</a>
               </td>
               <td>
-                <a href="https://w3id.org/security/v3-unstable">security-vocab-v3-unstable</a>
+                <a href="https://w3id.org/security/v3-unstable">https://w3id.org/security/v3-unstable</a>
```

Comment:
We should stop pointing to the v3-unstable security context across the board... we're moving to suites, we should make that change as rapidly as possible.

Reply:
If only I could have somehow broken those permanent links... folks would have no choice but to stop using them... sometimes a breaking change is needed to ensure a swift transition.... it you are not caching these things, you really ought not to be using them.

Reply:
Agreed - I'd like to completely remove v3-unstable usage and not register anything that's unstable going forward.